src/services/dashboards/dashboard-create/DashboardCreatePage.vue

```diff
@@ -84,12 +86,16 @@ export default {
 
         const handleClickCreate = () => {
             const dashboardCreateParams = {
-                dashboardScope: state.dashboardScope,
-                dashboardProject: state.dashboardScope === DASHBOARD_SCOPE.DOMAIN ? '' : dashboardProject.value,
-                dashboardTemplate: dashboardTemplate.value,
-                dashboardViewerType: state.dashboardViewerType,
+                name: 'name??????',
+                viewers: state.dashboardViewerType,
             };
-            console.log(dashboardCreateParams);
+
+            if (state.dashboardScope === DASHBOARD_SCOPE.PROJECT) {
+                SpaceConnector.clientV2.dashboard.projectDashboard.create({
+                    ...dashboardCreateParams,
+                    project_id: dashboardProject.value?.id ?? '',
+                });
+            } else SpaceConnector.clientV2.dashboard.domainDashboard.create(dashboardCreateParams);
```

Comment:
I think it'd be good if there was try catch!

Reply:
I forgot it .. Thanks a lot !